core/lib/zksync_core/src/lib.rs

```diff
@@ -235,6 +236,8 @@ pub enum Component {
     ProofDataHandler,
     /// Native Token fetcher
     NativeTokenFetcher,
+    /// Conversion rate API, for local development.
+    ConversionRateApi,
```

Comment:
I think it makes sense to prepend it with `Dev` here and everywhere else to clearly indicate that it's a local development utility and not something that is needed for production.

Reply:
Same as above